utils/Constants.ts

```diff
@@ -228,5 +233,7 @@ export const BASE_STAKING_MULTIPLIER = 1e6;
 export const chainIds = {
   mainnetFork: 1337,
   rsk: 30,
-  arbitrumTestnet: 421611
+  arbitrumTestnet: 421611,
```

Comment:
@kazazor do we add these also to the no-timelock recorder?

Reply:
Yes